.github/workflows/parachain.yml

```diff
@@ -145,6 +135,7 @@ jobs:
           --manifest-path cumulus/Cargo.toml
           --release --verbose
           --package bridge-hub-rococo-runtime
+          --features runtime-benchmarks  
```

Comment:
Nice!

Reply:
Yes, check `bridge-hub-rococo-runtime` with feature `runtime-benchmarks` to avoid the error from now on.